static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -348,10 +351,47 @@ define([
                 bonzo(player.el()).addClass(endState);
             });
         });
+
         player.on('playing', function () {
             bonzo(player.el()).removeClass(endState);
         });
     }
+    
+    function initFader(player){
+        var fader = $('.video-fader');
+        var faderTimeout;
+        function setFaderTimeout(){
+            fader.removeClass('video-fader--active');
+            clearTimeout(faderTimeout);
+            faderTimeout = setTimeout(function() {
+            fader.addClass('video-fader--active');
+            }, 3000);
+        }
+        function removeFadeScroll(){
+            var related = $('.content-footer').offset().top;
+            var scroll = $(window).scrollTop();
+            var position = Math.floor(related - scroll);
+                if (position <= 200){
+                    fader.removeClass('video-fader--active');
+                    clearTimeout(faderTimeout);
+                    bean.off(document.body, 'mousemove', setFaderTimeout);
+                } else if (!player.paused()){
+                    fader.addClass('video-fader--active');
+                    bean.on(document.body, 'mousemove', setFaderTimeout);
+                }
+        }
```

Comment:
I don't entirely understand the need for this function. Could you try to explain why it's needed?